pyproject.toml

```diff
@@ -39,6 +39,7 @@ dev = [
   "pytest==7.2.0",
   "syrupy==4.0.2",
   "tox==4.5.2",
+  "requests-mock==1.10.0"
```

Comment:
Can we use it? 🤔

Reply:
I don`t now, now i switched to unittest patch